manual/mapper/entities/README.md

```diff
@@ -159,6 +159,67 @@ private int day;
 This information is used by some of the DAO method annotations; for example,
 [@Select](../daos/select/)'s default behavior is to generate a selection by primary key.
 
+#### Computed properties
+
+Annotating an entity property with [@Computed] indicates that when retrieving data with the mapper
+this property should be set to the result of a computation on the Cassandra side, typically a
+function call:
+
+```java
+private int v;
+
+@Computed("writetime(v)")
+private long writetime;
+```
+
+The CQL return type of the formula must match the type of the property, otherwise an exception
+will be thrown.
+
+[@Computed] does not support case-sensitivity. If the expression contains case-sensitive column
+or function names, you'll have to escape them:
+
+```java
+@Computed("\"myFunction\"(\"myColumn\")")
+private int f;
+```
+
+[@Computed] fields are only used for queries, so they will not be considered for [@Update] or
+[@Insert] operations.
+
+Also note that like all other properties, the expected name in a query result for a [@Computed]
+property is based on the property name and the employed [@NamingStrategy](#naming-strategy). You may
+override this behavior using [@CqlName](#user-provided-names).
+
+Mapping computed results to property names is accomplished using [aliases].  If you wish to use
+entities with [@Computed] properties with [@GetEntity] or [@Query]-annotated dao methods, you
+must also do the same:
+
+```java
+@Entity
+class MyEntity {
+  @PartitionKey private int k;
+
+  private int v;
+
+  @Computed("ttl(v)")
+  private int myTtl;
+
+  @Computed("writetime(v)")
```

Comment:
Wouldn't `@Computed(expression = "writetime(v)", alias = "ts")` be more explicit?
If not: wouldn't `@Alias` be a better name than `@CqlName`?

Reply:
> `@Computed(expression = "writetime(v)", alias = "ts")`

This introduces a special case though, isn't it better if computed values behave like any other property?

> wouldn't `@Alias` be a better name than `@CqlName`?

👍 to that. This is in line with the renaming I suggested on `PropertyDefinition`.

Reply:
> Isn't it better if computed values behave like any other property?

That's what I was aiming for by using `@CqlName`.  `CqlName` is used to indicate the name of the cql column, but also now the aliased result name for computed values.  I think it fits really well.

If we change it to `@Alias`, the intent could be interpreted differently, effectively as a rename, but that will not generally be the case (only for `@Computed` properties).  I don't think there would be too much of a use case for adding something like `@Alias` to rename how to get at the resulting value beyond computed values.

Reply:
> If we change it to `@Alias`, the intent could be interpreted differently

Mmm that's a good point, "alias" would imply that we use a different name than the column's, but for non-computed columns that is not the case.
So I change my vote to keep `CqlName`. And we should also probably keep that name in `PropertyDefinition` (but I'm still in favor of `getSelector` for the other one).

Reply:
I'm sorry but I find `CqlName` way too vague. If I follow you correctly this annotation is being used to:

1. Customize regular column names: why not introduce a special `@Column` annotation?
2. To define aliases for computed properties: why don't we revert then to my initial idea of `@Computed(expression = "writetime(v)", alias = "ts")`? It seems that only computed properties will ever need aliases, so that option sounds fair to me.

Reply:
> why not introduce a special `@Column` annotation?

That sounds even more vague to me. Marking something as a "column" doesn't say anything about naming, and you might have other properties that are not annotated but will still map to a column.

> To define aliases for computed properties

The advantage of the current solution is that it works the same way whether the property is computed or not:

"What name should I use to get the property on my `Row`?"
* if `CqlName` is present, that
* otherwise, the name of the Java property, translated with the naming strategy

Why make it different for computed properties?

Reply:
You know what, I'm retiring myself from this endless debate and let you guys decide what is best. Good luck ;-)